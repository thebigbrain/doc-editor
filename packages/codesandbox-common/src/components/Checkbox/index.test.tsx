import 'jest-styled-components';
import React from 'react';
import mountWithTheme from '../../test/themeMount';
import { Checkbox } from '.';

describe('<Checkbox /> rendering', () => {
  it('no props', () => {
    const wrapper = mountWithTheme(<Checkbox />);
    expect(wrapper).toMatchSnapshot();
  });
  it('checkbox checked', () => {
    const wrapper = mountWithTheme(<Checkbox checked />);
    expect(wrapper).toMatchSnapshot();
  });
});
