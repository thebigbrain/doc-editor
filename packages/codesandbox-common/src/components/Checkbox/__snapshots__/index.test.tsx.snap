// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkbox /> rendering checkbox checked 1`] = `ReactWrapper {}`;

exports[`<Checkbox /> rendering no props 1`] = `ReactWrapper {}`;
